package.json

```diff
@@ -74,7 +74,8 @@
   },
   "dependencies": {
     "microdiff": "^1.3.1",
-    "p5": "^1.5.0"
+    "p5": "^1.5.0",
+    "tone": "^14.7.77"
```

Comment:
tone.js is not a dependency of the library, it should not be here but instead in dev-dependencies since you only want it for the example.